packages/test/test-end-to-end-tests/src/test/gc/gcReferenceUpdatesInSummarizer.spec.ts

```diff
@@ -79,37 +76,27 @@ class TestDataObject extends DataObject {
  * The difference between these tests and the ones in the file 'gcReferenceUpdatesInLocalSummary' is that here we submit
  * summaries to the server, load new containers from the summary downloaded from server and validate them.
  */
-describeFullCompat("GC reference updates in summarizer", (getTestObjectProvider) => {
+describeNoCompat("GC reference updates in summarizer", (getTestObjectProvider) => {
     let provider: ITestObjectProvider;
-    const factory = new DataObjectFactory(
+    const dataStoreFactory = new DataObjectFactory(
         "TestDataObject",
         TestDataObject,
         [SharedMatrix.getFactory(), SharedString.getFactory()],
         []);
 
-    const IdleDetectionTime = 100;
-    const summaryConfigOverrides: ISummaryConfiguration = {
-        ...DefaultSummaryConfiguration,
-        ...{
-            idleTime: IdleDetectionTime,
-            maxTime: IdleDetectionTime * 12,
-            initialSummarizerDelayMs: 10,
-        },
-    };
     const runtimeOptions: IContainerRuntimeOptions = {
         summaryOptions: {
-            summaryConfigOverrides,
-        },
-        gcOptions: {
-            gcAllowed: true,
+            disableSummaries: true,
```

Comment:
>disableSummaries: true, [](http://example.com/codeflow?start=12&length=23)

same as before - I don't think we need it  disableSummaries is deprecated and will go away.